canonical-websites/cn.ubuntu.com
# Add the 18.10 homepage and download links

## Done
Add the 18.10 homepage and download links

## QA
- Check that the homepage has the 18.10 takeover
- Go to /desktop and see the 18.10 button
